executor/common_linux.h

```diff
@@ -261,13 +268,20 @@ static void initialize_netdevices(void)
 				  "veth0_to_bridge", "veth1_to_bridge",
 				  "veth0_to_bond", "veth1_to_bond",
 				  "veth0_to_team", "veth1_to_team"};
+#endif
+#if SYZ_EXECUTOR || SYZ_ENABLE_NETDEV_DEVMASTERS
```

Comment:
Not sure if it makes sense to do DEVMASTERS and DEVTYPES if we have DEVNAMES disabled. We will just create some devices, but don't setup them and don't enable.

Reply:
Therefore we're basically going down to two options, e.g. NETDEV_DEVICES and NETDEV_SLAVES?

Reply:
DEVNAMES makes sense without DEVMASTERS and DEVTYPES, because there are some pre-created devices. So I think we still need 3, but just don't do DEVMASTERS if DEVNAMES is not enabled.